src/services/ui/src/pages/actions/WithdrawPackage.tsx

```diff
@@ -84,162 +85,83 @@ const WithdrawPackageForm: React.FC = ({ item }: { item?: ItemResult }) => {
-          onAccept={() => {
-            setSuccessModalIsOpen(false);
-            navigate({ path: "/details", query: { id } });
-          }}
-          onCancel={() => setSuccessModalIsOpen(false)} // Should be made optional
-          title="Withdraw Successful"
-          body={
-            <p>
-              Please be aware that it may take up to a minute for your status to
-              change on the Dashboard and Details pages.
-            </p>
-          }
-          cancelButtonVisible={false}
-          acceptButtonText="Go to Package Details"
-        />
-        {/* Error Modal */}
-        <ConfirmationModal
-          open={errorModalIsOpen}
-          onAccept={() => {
-            setErrorModalIsOpen(false);
-            navigate({ path: "/details", query: { id } });
-          }}
-          onCancel={() => setErrorModalIsOpen(false)}
-          title="Submission Error"
-          body={
-            <p>
-              An error occurred during issue.
-              <br />
-              You may close this window and try again, however, this likely
-              requires support.
-              <br />
-              <br />
-              Please contact the{" "}
-              <a
-                href="mailto:OneMAC_Helpdesk@cms.hhs.gov"
-                className="text-blue-500"
+            />
+            {errorMessage && (
+              <div className="text-red-500 mt-4">{errorMessage}</div>
+            )}
+            <div className="flex gap-2 my-8">
+              <Button type="submit">Submit</Button>
+              <Button
+                onClick={() => setCancelModalOpen(true)}
+                variant="outline"
               >
-                helpdesk
-              </a>{" "}
-              . You may include the following in your support request: <br />
-              <br />
-              <ul>
-                <li>SPA ID: {id}</li>
-                <li>Timestamp: {Date.now()}</li>
-              </ul>
-            </p>
-          }
-          cancelButtonVisible={true}
-          cancelButtonText="Return to Form"
-          acceptButtonText="Exit to Package Details"
-        />
-        {/* Cancel Modal */}
-        <ConfirmationModal
-          open={cancelModalIsOpen}
-          onAccept={() => {
-            setCancelModalIsOpen(false);
-            navigate({ path: "/details", query: { id } });
-          }}
-          onCancel={() => setCancelModalIsOpen(false)}
-          cancelButtonText="Return to Form"
-          acceptButtonText="Leave Page"
-          title="Are you sure you want to cancel?"
-          body={
-            <p>
-              If you leave this page you will lose your progress on this form
-            </p>
-          }
-        />
+                Cancel
+              </Button>
+            </div>
+          </form>
+        </I.Form>
       </div>
     </>
   );
 };
-
-export const WithdrawPackage = () => (
-  <PackageActionForm>
-    <WithdrawPackageForm />
-  </PackageActionForm>
```

Comment:
BEGONE ye ol' dependency -- we no longer need to wrap these as the responsibilities handled by this are now handled in `ActionFormIndex` (breadcrumbs and page container), and `ActionFormSwitch` (item fetching and piping as props).